hw3/Task_1.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+//Task 1
```

Comment:
Could you please add description of the task at the top of the each file.

Reply:
Should I copy the text of each task in Russian and make it commented?

Reply:
Done.